app/models/shipit/user.rb

```diff
@@ -123,5 +129,20 @@ def identify_renamed_user!
     rescue Octokit::NotFound
       false
     end
+
+    def org_email?(email_address)
+      return false if email_address.blank?
+
+      org_domains = Shipit.preferred_org_emails
+      return true if org_domains.empty? || org_domains.all?(&:blank?)
```

Comment:
Just `Shipit.preferred_org_emails.blank?` is enough. No need to be defensive about people putting blank strings in that array.